### PR TITLE
Finished homework

### PR DESCRIPTION
**Pull request by victorprogramm**

### CODE REVIEW

File: src/main/java/co/inventorsoft/academy/collections/model/Range.java [hunk 1]
```diff
@@ -1,60 +1,120 @@
...
 import java.util.Collection;
 import java.util.Iterator;
+import java.util.List;
 import java.util.Set;
+import java.util.function.Function;
 
 public class Range<T> implements Set<T> {
 
+
+    public List<T> MyList = new ArrayList<>();
```
Comment (george-stotle) [inline src/main/java/co/inventorsoft/academy/collections/model/Range.java]: In java we name fields starting with lowercase

File: src/main/java/co/inventorsoft/academy/collections/model/Range.java [hunk 2]
```diff
@@ -1,60 +1,120 @@
...
+        @Override
+        public Integer apply (Integer y) {
+            return y + 1;
+        }
 
     }
-}
+    public static Range<Integer> of(Integer first, Integer last) {
+        MyInteger isA = new MyInteger();
+        Range<Integer> intRange = Range.of(first, last, isA);
```
Comment (george-stotle) [inline src/main/java/co/inventorsoft/academy/collections/model/Range.java:85]: use lambda instead of creating a class

**Review by george-stotle — CHANGES_REQUESTED** [review]

File: src/main/java/co/inventorsoft/academy/collections/model/Range.java (hunk 1 above)
Comment (victorprogramm) [reply src/main/java/co/inventorsoft/academy/collections/model/Range.java]: Changed fields to lowercase

File: src/main/java/co/inventorsoft/academy/collections/model/Range.java [hunk 3]
```diff
@@ -1,60 +1,120 @@
...
+    public static class MyFloat implements Function<Float, Float> {
+
+        @Override
+        public Float apply(Float f) {
+            return f + 0.1f;
+        }
+    }
+    public static Range<Float> of(Float first, Float last) {
+        Function<Float, Float> flt = new MyFloat();
+        Range<Float> floatRange = Range.of(first, last, flt);
```
Comment (george-stotle) [inline src/main/java/co/inventorsoft/academy/collections/model/Range.java:100]: pass lambda here instead of creating MyFloat

**Pull request closed without merging**